templates/basic-stage.yml

```diff
@@ -0,0 +1,21 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  labels:
+    provider: codeready-toolchain
+      name: basic-stage
+objects:
+- apiVersion: v1
+  kind: Namespace
+  metadata:
+    annotations:
+      openshift.io/description: ${USERNAME}-stage
+      openshift.io/display-name: ${USERNAME}-stage
+      openshift.io/requester: ${USERNAME}
+    labels:
+      provider: codeready-toolchain
+          name: ${USERNAME}
```

Comment:
```suggestion
    name: ${USERNAME}
```

Reply:
Yep. Need to fix formatting but also the namespace name. See my suggestion below.